### ISSUE TITLE
Update method

### ISSUE THREAD

**Issue by prasant10050** [chuyentt/localstore#9]
How to update existing stored data against key

**Comment by nathanael540** [comment]
Yes, i need know too. I tried deleting and then set up again, but sometimes it isn't working properly.

What the best way to update?

**Comment by HannesGitH** [comment]
isnt `....doc(id).set(json);` updating the document with `id` to the new json?
